src/main/java/org/mskcc/cbio/oncokb/service/impl/TokenStatsServiceImpl.java

```diff
@@ -62,6 +62,10 @@ public void delete(Long id) {
         tokenStatsRepository.deleteById(id);
     }
 
+    public void clearTokenStats() {
+        tokenStatsRepository.deleteAll();
```

Comment:
Shouldn't be deleting all but delete the tokenState before the timestamp you used in `updateTokenStats`